pretixprint/app/src/main/java/eu/pretix/pretixprint/ui/PrinterSetupActivity.kt

```diff
@@ -98,7 +99,13 @@ class PrinterSetupActivity : AppCompatActivity() {
                     manager.requestPermission(iminPrinter, permissionIntent)
                 }
             }
-            settingsStagingArea.put("hardware_${useCase}printer_mode", ESCPOS().identifier)
+            if (useCase == "receipt") {
+                settingsStagingArea.put("hardware_${useCase}printer_mode", ESCPOS().identifier)
+            } else {
+                settingsStagingArea.put("hardware_${useCase}printer_mode", GraphicESCPOS().identifier)
+                settingsStagingArea.put("hardware_${useCase}printer_graphicescposcompat", "true")
+                // FIXME: does this need dpi and page width params?
```

Comment:
Yes, dpi and page width seem relevant to me. 200dpi and 72mm looks fine to me, but we should set it here in case one had something else configured previously